.github/actions/docker-build/action.yml

```diff
@@ -62,10 +55,6 @@ runs:
         echo "${image_digest}"
         echo "DIGEST=${{ inputs.docker-registry }}/${{ inputs.component }}@${image_digest}" >> "$GITHUB_OUTPUT"
 
-    # Delete older docker images to prevent cache bloat.
-    - run: bin/docker-cache-prune
```

Comment:
Could we also remove the script from the file tree? I don't think we're using it anywhere else.

```
:; rg 'docker-cache-prune'
actions/docker-build/action.yml
66:    - run: bin/docker-cache-prune
```

Can also be done as a follow-up. It's always dangerous when removing (seemingly) unused scripts.

Reply:
i did consider removing this script, along with references to `DOCKER_BUILDKIT_CACHE`, but i wasn't sure if anyone was using these for their own local development. happy to remove all this if not the case.